### PR TITLE
Added mailchimp signup form

### PR DESCRIPTION
**Pull request by bc-AlyssNoland**
#### What?

See title. Also removed toc footers in place of this form. 

#### How To Test

Verify email is sent to mailchimp when using form

#### Screenshots (if appropriate)

pasted image at 2017_02_08 10_57 am

### CODE REVIEW

File: source/layouts/_mailchimp.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!-- Begin MailChimp Signup Form -->
+<div id="mc_embed_signup">
+<form action="//bigcommerce.us13.list-manage.com/subscribe/post?u=b9c3e5c67ccfd920368b5967b&amp;id=7a3d485661" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+  <div id="mc_embed_signup_scroll">
+    <h5 style="color: #4E75F8">Subscribe to developer updates</h5>
```
Comment (tekstrand) [inline source/layouts/_mailchimp.html.erb]: any way you can use a class here instead?

**Comment by tekstrand** [comment]
LGTM other than the one comment I made

File: source/layouts/_mailchimp.html.erb (hunk 1 above)
Comment (bc-AlyssNoland) [inline source/layouts/_mailchimp.html.erb]: @tekstrand yep, I'll make that change.

**Comment by bc-AlyssNoland** [comment]
@tekstrand Added a class at that comment and made use of the existing variable for that color. 👍

**Pull request merged**